tests/unit_tests/scheduler/test_openpbs_driver.py

```diff
@@ -23,7 +23,11 @@
 )
 
 
-@given(st.lists(st.sampled_from(["H", "Q", "R", "F"])))
+@given(
+    st.lists(
+        st.sampled_from(["B", "E", "F", "H", "M", "Q", "R", "S", "T", "U", "W", "X"])
```

Comment:
You can use the list in `JobState` type rather than copying it (single source of truth and all that)

```suggestion
        st.sampled_from(JobState.__args__)
```

Reply:
Perfect, I was looking for it, but a flaky and chatty friend lied to me.